test/extended/testdata/builds/cluster-config/registry-whitelist.yaml

```diff
@@ -6,3 +6,5 @@ spec:
   registrySources:
       allowedRegistries:
       - quay.io
+      - registry.redhat.io
+      - registry.access.redhat.com
```

Comment:
As discussed in slack with @runcom and @deads2k - this can be a dangerous feature if the release payload repo isn't included in this list. If the payload repo is not allowed, the cluster dies on the MCO rollout.

Reply:
I guess we can inject the payload repo here but that was a feature  with docker as well and I guess it's use at your own risk as well. Now, the cluster commits suicide but if I manually fix the broken master, does the etcd get quorum again reconciling and cluster is back up again? cause that's the case, I would still think it's still "hey, this is root stuff, be careful in using it" (as it was with docker anyway)